### PR TITLE
Changes to readme and documentation

### PR DESCRIPTION
**Pull request by leonardovida**
As from title, I began correcting few mistakes and inaccuracies in the documentation and readme.

@vankesteren @J535D165 do you know how to link between vignettes? E.g. if I want to create a link from one vignette to another. Using the path to the Rmd does not work.

### CODE REVIEW

**Comment by leonardovida** [comment]
@vankesteren @J535D165 This PR is finished:
* Documentation fixed and corrections
* Addition of two tutorials
* Small addition of `reduce_fun`

Let me know when I can merge!

File: README.md [hunk 1]
```diff
@@ -146,8 +157,8 @@ Contributions are what make the open source community an amazing place to
 learn, inspire, and create. Any contributions you make are **greatly
 appreciated**.
 
-In this project, the [Gitflow workflow](https://nvie.com/posts/a-successful-
-git-branching-model/) is used. Instead of having a single `master`/`main`
+In this project, the [Gitflow workflow](https://nvie.com/posts/a-successful-git-branching-model/)
+is used. Instead of having a single `master`/`main`
 branch, the project makes use of two branches to record the history of the
 project: `develop` and `master`. The `master` branch is used only for the
```
Comment (vankesteren) [inline README.md]: `master` -> `main`

**Review by vankesteren — APPROVED** [review]
Let's merge and I'll look at the vignettes after the merge!

**Comment by J535D165** [comment]
Nice contribution!

**Pull request merged**